client/my-sites/exporter/export-media-card/index.js

```diff
@@ -61,6 +67,7 @@ class ExportMediaCard extends Component {
 export default connect(
 	state => ( {
 		mediaExportUrl: getMediaExportUrl( state ),
+		mediaStorageUsed: getMediaStorageUsed( state, getSelectedSiteId( state ) ),
```

Comment:
Nitpick: since `mediaStorageUsed` is used only for setting `hasNoMediaFiles`, I think we can expose a `hasNoMediaFiles` prop directly here.

 ```suggestion
		hasNoMediaFiles: getMediaStorageUsed( state, getSelectedSiteId( state ) ) === 0 && ! getMediaExportUrl( state ),
```

Reply:
Yep, done! :)